src/pip/_internal/commands/install.py

```diff
@@ -406,7 +407,8 @@ def run(self, options, args):
             if options.target_dir or options.prefix_path:
                 warn_script_location = False
 
-            content_addressable_pool = "content-addressable-pool" in options.features_enabled
+            content_addressable_pool = "content-addressable-pool" \
+                                       in options.features_enabled
```

Comment:
minor/optional: so far i've seen that they prefer to parenthesize and then let the autoformatter do the alignment over backslashes:

```suggestion
            content_addressable_pool = ("content-addressable-pool" 
                                        in options.features_enabled)
```